Makefile.uk

```diff
@@ -41,6 +41,14 @@ $(error Require libunwind)
 endif
 endif
 
+CLANG_CHECK = 0
+cc-name = $(shell $(CC) -v 2>&1 | grep -q "clang version" && echo clang || echo gcc)
+ifeq ($(cc-name), clang)
+CLANG_CHECK = 1
+endif
+export CLANG_CHECK
```

Comment:
Checking if `clang` is enabled is already done by [this function](https://github.com/unikraft/unikraft/blob/staging/support/build/Makefile.rules#L37-L40). You can use it like `ifeq ($(call have_clang),y)`.

Reply:
OK, so this is not needed, I've removed it.